src/render/prettify.js

```diff
@@ -4,10 +4,13 @@ const babelParser = require('prettier/parser-babel')
 
 function prettify(raw) {
   return prettier.format(raw, {
-    semi: true,
+    semi: false,
     arrowParens: 'avoid',
     parser: 'babel',
     plugins: [babelParser],
+    singleQuote: true,
+    trailingComma: 'es5',
+    printWidth: 100,
```

Comment:
This is technically a breaking change, as it means that the output will produce a different output from before. What was the reasoning behind it?

Reply:
Produced k6 script should look more like examples on k6.io.